index.html

```diff
@@ -53,5 +53,209 @@ <h3 class="hero-highlight-2">I’am a Software Developer</h3>
+      </article>
+      <article>
+        <div class="cards">
+          <div class="card-content">
+            <h2 class="cd-header">profesional art printing data</h2>
+            <p class="cd-txt">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-tool">
+              <li class="tools tools-spec">html</li>
+
+              <li class="tools tools-spec bootstrap">bootstrap</li>
+
+              <li class="tools tools-spec">Ruby</li>
+            </ul>
+            <button type="button" class="button btn">see project</button>
+          </div>
+        </div>
+      </article>
+      <article>
+        <div class="cards">
+          <div class="card-content">
+            <h2 class="cd-header">profesional art printing data</h2>
+            <p class="cd-txt">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-tool">
+              <li class="tools tools-spec">html</li>
+
+              <li class="tools tools-spec bootstrap">bootstrap</li>
+
+              <li class="tools tools-spec">Ruby</li>
+            </ul>
+            <button type="button" class="button btn">see project</button>
+          </div>
+        </div>
+      </article>
+      <article>
+        <div class="cards">
+          <div class="card-content">
+            <h2 class="cd-header">profesional art printing data</h2>
+            <p class="cd-txt">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-tool">
+              <li class="tools tools-spec">html</li>
+
+              <li class="tools tools-spec bootstrap">bootstrap</li>
+
+              <li class="tools tools-spec">Ruby</li>
+            </ul>
+            <button type="button" class="button btn">see project</button>
+          </div>
+        </div>
+      </article>
+      <article>
+        <div class="cards">
+          <div class="card-content">
+            <h2 class="cd-header">profesional art printing data</h2>
+            <p class="cd-txt">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-tool">
+              <li class="tools tools-spec">html</li>
+
+              <li class="tools tools-spec bootstrap">bootstrap</li>
+
+              <li class="tools tools-spec">Ruby</li>
+            </ul>
+            <button type="button" class="button btn">see project</button>
+          </div>
+        </div>
+      </article>
+    </section>
+    <section class="about-section">
+      <div class="about-header">
+        <h1>About me</h1>
```

Comment:
- [ ] As we have used the `h1` tag as heading and it can only once on a single page, so kindly change it to another `hx` tag.